extensions/azurecore/package.json

```diff
@@ -128,6 +128,156 @@
               "Microsoft Authentication Library"
             ],
             "deprecationMessage": "Warning: ADAL has been deprecated, and is scheduled to be removed in a future release. Please use MSAL (default option) instead."
+          },
+          "azure.providerSettingsJson": {
+            "type": "array",
+            "description": "%config.providerSettingsJson%",
+            "scope": "resource",
+            "default": [
+              {
+                "name": "Default",
+                "settings": {
+                  "metadata": {
+                    "displayName": "DefaultCloud",
```

Comment:
I don't think you need to specify default value for this setting - as it shows up in Linked Account view, even when nothing is configured.

Reply:
Noticed this still exists..